src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -382,31 +388,43 @@ private function process_directives_args( string $html, array &$context_stack, a
 				}
 			}
 		}
-
 		/*
 		 * It returns null if the HTML is unbalanced because unbalanced HTML is
 		 * not safe to process. In that case, the Interactivity API runtime will
-		 * update the HTML on the client side during the hydration.
+		 * update the HTML on the client side during the hydration. It will also
+		 * display a notice to the developer to inform them about the issue.
 		 */
-		return $unbalanced || 0 < count( $tag_stack ) ? null : $p->get_updated_html();
+		if ( $unbalanced || 0 < count( $tag_stack ) ) {
+			$tag_errored = 0 < count( $tag_stack ) ? end( $tag_stack )[0] : $tag_name;
+			/* translators: %1s: Namespace processed , %2s: The tag that caused the error; could be any HTML tag.  */
```

Comment:
```suggestion
			/* translators: %1s: Namespace processed, %2s: The tag that caused the error; could be any HTML tag.  */
```

Reply:
Done in https://github.com/WordPress/wordpress-develop/pull/6413/commits/3e64ebb6a8e94a13f937d221e110633167d61af0